subsys/bluetooth/audio/Kconfig.tbs

```
@@ -0,0 +1,155 @@
+# Bluetooth Audio - Call control configuration options
+#
+# Copyright (c) 2020 Bose Corporation
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+if BT_AUDIO
+
+#####################  Telephone Bearer Service #####################
+
+config BT_TBS
```

Should this be a menuconfig instead, in order to give a better grouping in `menuconfig` with a subpage for all BT_TBS related settings

```suggestion
menuconfig BT_TBS
```